internal/coreinternal/golden/testdata/standardize-metrics/after.yaml

```diff
@@ -0,0 +1,183 @@
+resourceMetrics:
+  - resource:
+      attributes:
+        - key: a
+          value:
+            stringValue: AAAA
+    scopeMetrics:
+      - metrics:
+          - description: Yet another fake metric for test
+            name: storage.amplitude
+            sum:
+              aggregationTemporality: 2
+              dataPoints:
+                - asInt: "30"
+                  attributes:
+                    - key: a
+                      value:
+                        stringValue: AAAA
+                  timeUnixNano: "2000000"
+                - asInt: "10"
+                  attributes:
+                    - key: a
+                      value:
+                        stringValue: AAAA
+                  timeUnixNano: "1000000"
+                - asInt: "30"
+                  attributes:
+                    - key: a
+                      value:
+                        stringValue: AAAA
+                  timeUnixNano: "1000000"
+                - asInt: "0"
+                  attributes:
+                    - key: a
+                      value:
+                        stringValue: BBBB
+                  timeUnixNano: "1000000"
+                - asInt: "0"
+                  attributes:
+                    - key: d
+                      value:
+                        stringValue: AAAA
+                  timeUnixNano: "1000000"
+            unit: "1"
+        scope:
+          attributes:
+            - key: a
+              value:
+                stringValue: AAA
+          name: otelcol/FakeMetric
+          version: latest
+      - metrics:
+          - description: Fake Metric
+            name: storage.amplitude
+            sum:
+              aggregationTemporality: 2
+              dataPoints:
+                - asInt: "0"
+                  attributes:
+                    - key: a
+                      value:
+                        stringValue: AAAA
+                  timeUnixNano: "1000000"
+                - asInt: "0"
+                  attributes:
+                    - key: a
+                      value:
+                        stringValue: AAAA
+                  timeUnixNano: "1000000"
```

Comment:
Something appears to be wrong here because we have two data points with the same attributes and timestamps.

Reply:
This is because the original for those
![image](https://github.com/open-telemetry/opentelemetry-collector-contrib/assets/38274348/b1509d62-5729-4612-859f-dff715959444)
are both the same timeUnixNano. I was under the understanding that in order for the timestamps to scale up, they must
1. have the same attribute
2. have timestamps that differ from each other.
Would it never be the case that in a time series where the attributes are the same for a metric, the times stamps are also the same. Or in other words would you ever have a situation like the picture above, where within a metric you have datapoints of the same attributes and same timestamp?

Reply:
> would you ever have a situation like the picture above, where within a metric you have datapoints of the same attributes and same timestamp?

I don't believe this is valid, for sums and gauges at least. It's not strictly prohibited, but I think we don't need to handle it here.